ansible/test_settings.yml

```diff
@@ -1,37 +1,62 @@
-# Debug mode (Will print out extra debug information. Only enable if asked to by RedHat)
+# Debug mode (Will print out extra debug information. Only enable if asked
+# to by RedHat)
 redhat_debug_mode: False
 
-# Red Hat subscription username, password, pool_id used for accessing Red Hat repositories.  If you do not have one
-# please ask your Red Hat EPM to assist.
-rh_sub_mode: true
+# Red Hat subscription username, password, pool_id used for accessing Red
+# Hat repositories.  If you do not have one please ask your Red Hat EPM
+# to assist.
+rh_sub_mode: True
 rh_sub_username:
 rh_sub_pass:
 rh_sub_pool_id:
 
-# enable the below if internally testing for QE and use the command found at -> https://wiki.test.redhat.com/ctrautma
-# and search for the section Subscription manager testing and enclose it with ""
+# Enable the below if internally testing for QE and use the command found
+# at -> https://wiki.test.redhat.com/ctrautma and search for the section
+# Subscription manager testing and enclose it with ""
 # DO NOT COMMIT THIS FILE TO GITHUB if this change is made!!!!!
-qe_subscription_mode: false
+qe_subscription_mode: False
 qe_subscription_command:
 
-# Allows custom repo support if rh_sub or qe_subscription are not available.
-custom_repo_mode: false
-custom_repo_path: 
+# Alternate solution to subscription manager and QE subscription. Allows
+# custom repos to be used instead.  User is responsible for making sure
+# that the selected repo includes the required binaries.
+custom_repo_setup: False
+custom_repo_path:
 
+#######################################
+# T-Rex Server specific configuration #
+####################################### 
 # T-Rex version to use (example v2.53 : Note please include the v).
-# This is the version that will be downloaded from git and installed on the
-# T-Rex server.
+# This is the version that will be downloaded and installed on the T-Rex
+# server.
 trex_path: https://trex-tgn.cisco.com/trex/release
-trex_version: v2.53
+trex_version: v2.72
```

Comment:
Need to check with Eelco if his scripts are supporting version 2.72.  I believe at one point updating past this version was an issue.  Will double check.

Reply:
The update for the newer API was added previously, so if the test works the upgrade should be fine. It would be interesting to do the actual run with both versions to see if there is any performance degradation.